assets/js/search.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Javascript specially for the IaC Library page.
+ */
+
+(function () {
+
+  // Returns a function, that, as long as it continues to be invoked, will not
+  // be triggered. The function will be called after it stops being called for
+  // N milliseconds. If `immediate` is passed, trigger the function on the
+  // leading edge, instead of the trailing.
+  // From: https://davidwalsh.name/javascript-debounce-function
+  function debounce(func, wait, immediate) {
+    let timeout;
+    return function () {
+      let context = this,
+        args = arguments;
+      let later = function () {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      let callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
+    };
+  }
+
+  function showItemsCount(totalCount, numSubmodules) {
+    if (searchEntry.type == 'libraryEntries') {
+      if (totalCount > 0 && numSubmodules > 0) {
+        $('#search-results-count').html("<strong>" + totalCount + "</strong> repos (~<strong>" + numSubmodules + "</strong> modules)");
+      } else {
+        $('#search-results-count').text("0 repos");
+      }
+    } else {
+      if (totalCount > 0 && numSubmodules == 0) {
+        $('#search-results-count').html("<strong>" + totalCount + "</strong> found");
+      }
+    }
+  }
+
+  function displayFilterTags() {
+    if (searchEntry.type == 'guideEntries') {
+
+      //Filters the tags from the array of objects and flattens it out since it returns an array of arrays
+      let tags = searchEntry.entries.map(entry => entry.tags.split(',')).reduce((a, b) => a.concat(b), []);
+
+      tags.filter((tag, index) => {
+        if ((tags.indexOf(tag) === index) && (tag.length != "")) {
+          $('.tags').append(`<label class="checkbox"><input value=${tag} type="checkbox">${tag}</label>`);
```

Comment:
Filter tags look misaligned and are not properly capitalized 
<img width="816" alt="Screen Shot 2019-09-05 at 14 14 26" src="https://user-images.githubusercontent.com/1516418/64352701-67165f00-cff4-11e9-80b5-f53fe6abb776.png">